schema-registry/core/src/main/java/com/hortonworks/registries/schemaregistry/SchemaVersionLifecycleManager.java

```diff
@@ -675,6 +685,34 @@ public SchemaIdVersion mergeSchemaVersion(Long schemaVersionId, SchemaVersionMer
         return new ImmutablePair<>(context, schemaVersionLifecycleState);
     }
 
+    public SchemaVersionLifecycleContext createSchemaVersionLifeCycleContext(Long schemaVersionId, SchemaVersionLifecycleState schemaVersionLifecycleState) throws SchemaNotFoundException {
+        // get the current state from storage for the given versionID
```

Comment:
Can we file github issue for this to enhance storage manager? We already support `SearchQuery` with OrderBy, which can be enhanced to take LIMIT.